src/layers/generic.cpp

```diff
@@ -19,7 +19,7 @@ namespace marian {
   // This function assumes that the object holds one or more factor logits.
   // It applies the supplied loss function to each, and then returns the aggregate loss over all factors.
   Expr Logits::applyLossFunction(const Words& labels, const std::function<Expr(Expr/*logits*/, Expr/*indices*/)>& lossFn) const {
-    LOG_ONCE(info, "[logits] applyLossFunction() for {} factors", logits_.size());
+    LOG_ONCE(debug, "[logits] applyLossFunction() for {} factors", logits_.size());
```

Comment:
Let's keep "info" for now. This is a relatively new feature and good to monitor it.

Reply:
Done, and changed into a little less confusing "Applying loss function ..."